planning_launch/config/obstacle_stop_planner.yaml

```diff
@@ -0,0 +1,15 @@
+/**:
+  ros__parameters:
+    # stop planner
```

Comment:
Refer this.
https://github.com/tier4/AutowareArchitectureProposal.iv/blob/b0ba4f052ed836f5bc69e87d7f3ca83fec37f8e2/planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/config/obstacle_stop_planner.yaml#L3

Reply:
fixed